seqr/utils/file_utils.py

```diff
@@ -34,19 +38,20 @@ def file_iter(file_path, byte_range=None, raw_content=False):
     if _is_google_bucket_file_path(file_path):
         for line in _google_bucket_file_iter(file_path, byte_range=byte_range, raw_content=raw_content):
             yield line
+    elif byte_range:
+        command = 'dd skip={offset} count={size} bs=1 if={file_path}'.format(
+            offset=byte_range[0],
+            size=byte_range[1]-byte_range[0],
+            file_path=file_path,
+        )
+        process = _run_command(command)
+        for line in process.stdout:
+            yield line
     else:
         mode = 'rb' if raw_content else 'r'
         with open(file_path, mode) as f:
-            if byte_range:
-                f.seek(byte_range[0])
-                for line in f:
-                    if f.tell() < byte_range[1]:
-                        yield line
-                    else:
-                        break
-            else:
-                for line in f:
-                    yield line
+            for line in f:
```

Comment:
This seems like a suboptimal way to read binary files. Depending on the file, new-line chars may be very unevenly distributed in the file, so the "line" size will be very uneven. Reading a fixed number of bytes at a time should provide more consistent i/o speed.

Reply:
this hasn't changed as part of this PR though so I'm leaving it as-is for now

Reply:
It does return `lines` with large size differences. But, they will be used by `StreamingHttpResponse` as a byte stream in seqr's case. So, it seems doesn't matter.